api/model/api.go

```diff
@@ -319,13 +322,14 @@ type MethodTransformMeta struct {
 }
 
 type ValidatePathMeta struct {
-	// Allows override of default 422 Unprocessible Entity response code for validation errors.
+	// Allows override of default 422 Unprocessable Entity response code for validation errors.
 	ErrorResponseCode int        `json:"error_response_code"`
 	Path              string     `json:"path"`
 	Method            HttpMethod `json:"method"`
 	SchemaB64         string     `json:"schema_b64,omitempty"`
```

Comment:
@asoorm  Should we continue to support the SchemaB64 version after enabling Schema? Because the ValidateJSON field of ExtendedPathsSets was previously commented, now I uncommented it.

cc: @komalsukhani

Reply:
@buraksekili I may be wrong, but I seem to remember that we support both because one version is required for tyk with dashboard, and another one for Tyk CE, or maybe the Dashboard stores the payload B64 encoded inside Mongo. 

It's certainly worth double checking with @furkansenharputlu and @buger what makes sense and how this logic works from the dashboard and Gateway side. 

My initial thoughts are that we might need to support both inside the K8s CRD and leave it to the end-user to decide what they want to send us. Then the reconcile loop would be responsible for transforming the payload as required for the Gateway or Dashboard client. The reason for these initial thoughts is because in a dev environment, I would want to be able to export my API Definition AS IS (and regardless of whether CE or PRO), and I would want to be able to drop that in to a CRD without having to do any manual conversion.

Reply:
Thanks @asoorm, I will leave the `SchemaB64` as it is. 

Reply:
Since providing `schema` is enough for validating JSON, I removed `schema_b64` from ValidatePathMeta.